.github/CODEOWNERS

```diff
@@ -0,0 +1,12 @@
+# Each line is a file pattern followed by one or more owners.
+# For more on the CODEOWNERS file, please see:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @team-members will be requested for review when someone
+# opens a pull request.
+* @DataBiosphere/platform-foundation-codeowners
+
+# ensure that this repo's permissions can only be changed by CODEOWNERS
```

Comment:
This isn't about permissions. Isn't it just saying that the CODEOWNERS file requires review by platform-foundation-codeowners? And isn't that already covered by the default statement at line 9?

Reply:
AFAIK the presence of CODEOWNERS does not by itself enforce that they're required for a review. That requires setting the branch protections in the repo config. It also doesn't mean that they're tagged on reviews, this also must be enabled separately. My understanding is that this file is a configuration setting that _can_ be used but isn't' by default.

Also this line does nothing since this path is already included in `*` which is set above, so I would remove it.

Reply:
That's true, the number of reviews and way they are selected is done through branch protections in the repo config.

It is possible to override assignments because the last specified does win out, but since they are the same, I see no point in explicitly assigning CODEOWNERS to the group.

Reply:
All good! I can remove it. I was explicitly [calling it out](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-and-branch-protection) as a reflection of this section in the docs (see specifying CODEOWNERS specifically).

Reply:
If your repo is set up to require reviews by CODEOWNERS, it's true that having the explicit assignment to the CODEOWNERS file is the most secure because it guarantees that all edits to the file are reviewed by the group. But I don't think we really need that-- this isn't so much about locking down the repo as it as about helping ensure folks know who to contact. And if you have multiple teams contributing to the repo (like terra-ui), you expect CODEOWNERS to be edited by multiple teams. https://github.com/DataBiosphere/terra-ui/blob/dev/.github/CODEOWNERS